src/main/java/io/stargate/sgv2/jsonapi/service/schema/collections/CollectionSchemaObject.java

```diff
@@ -249,13 +166,14 @@ public static CollectionSchemaObject getCollectionSettings(
       }
       final String comment = (String) table.getOptions().get(CqlIdentifier.fromInternal("comment"));
       return createCollectionSettings(
-          keyspaceName, collectionName, true, vectorSize, function, comment, objectMapper);
+          keyspaceName, collectionName, table, true, vectorSize, function, comment, objectMapper);
```

Comment:
If we are passing "table", are `keyspaceName` and `collectionName` still needed?

Reply:
was worried about unpicking a lot of logic for the different ways we create, added https://github.com/stargate/data-api/issues/1459 to make it better